roles/molecule-test-splitter/library/molecule_splitter.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python
```

Comment:
I'd prefer not to create a 2nd splitter role. I would much rather see us refactor ansible-test-splitter into something more generic.

Reply:
trying to merge this splitter with the amazon one will increase the complexity of the role, this is just a simple executable listing all available scenario.

Reply:
yes, that is true. However, highlights exactly why we should use a single code base.

We really shouldn't need to dynamically create child jobs all the time, lets get into a 1:1 and sync on this to better understand what troubles you are dealing with.